### PR TITLE
Schema fixes

### PR DESCRIPTION
**Pull request by BenjaminMoe**
Addresses: https://github.com/w3c-ccg/traceability-vocab/issues/560

Screenshot 2022-10-03 at 23-15-27 Traceability Vocabulary Test Suite

Added Jest tests to CI to check schemas on PR.

### CODE REVIEW

**Review by nissimsan — APPROVED** [review]

File: docs/openapi/components/schemas/common/AgricultureProduct.yml [hunk 1]
```diff
@@ -108,27 +108,19 @@ properties:
...
-          "description": "Organic fruits & vegetables",
-          "email": "Chloe.Ondricka42@jimbofarm.org",
-          "phoneNumber": "555-496-7149",
-          "faxNumber": "555-209-6489"
-        },
-        "jobTitle": "District Quality Associate"
+        "type": ["Organization"],
+        "name": "Jimbo's Awesome Farm",
+        "description": "Organic fruits & vegetables",
+        "email": "Chloe.Ondricka42@jimbofarm.org",
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/AgricultureProduct.yml:121]: I'll grant you, `jimbofarm.org` doesn't exist today, but that doesn't guarantee it never will, nor that this sample data won't lead to unknowable problems if that domain is ever live. Still, please, let's always use `.example` or `.example.org` or the like for TLDs in sample data. `jimbofarm.example` or ``jimbofarm.example.org` are both viable changes.
```suggestion
        "email": "Chloe.Ondricka42@jimbofarm.example.org",
```

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/credentials/FSMAShippingCTECertificate.yml [hunk 2]
```diff
@@ -94,13 +102,13 @@ example: |-
...
-        "type": "FSMAShipment",
+        "type": ["FSMAShipment"],
         "product": {
-          "type": "FSMAProduct",
+          "type": ["FSMAProduct"],
           "traceabilityLot": {
-            "type": "FSMATraceabilityLot",
+            "type": ["FSMATraceabilityLot"],
             "lotCode": "CHE-MIG-TTF1-061321-H37J",
             "lotCodeAssignmentMethod": "Lot codes are assigned based on the commodity, variety, location, harvest date and a random 4 digit string.",
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/FSMAShippingCTECertificate.yml:113]:
```suggestion
            "lotCodeAssignmentMethod": "Lot codes are assigned based on the commodity, variety, location, harvest date, and a random 4 digit string.",
```

File: docs/openapi/components/schemas/credentials/FSMATransformingCTECertificate.yml [hunk 3]
```diff
@@ -94,11 +102,11 @@ example: |-
...
-      "type": "FSMATransformingCTE",
+      "type": ["FSMATransformingCTE"],
       "foodUsed": {
-        "type": "FSMAProduct",
+        "type": ["FSMAProduct"],
         "traceabilityLot": {
-          "type": "FSMATraceabilityLot",
+          "type": ["FSMATraceabilityLot"],
           "lotCode": "CHE-MIG-TTF1-061321-H37J",
           "lotCodeAssignmentMethod": "Lot codes are assigned based on the commodity, variety, location, harvest date and a random 4 digit string.",
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/FSMATransformingCTECertificate.yml:111]:
```suggestion
          "lotCodeAssignmentMethod": "Lot codes are assigned based on the commodity, variety, location, harvest date, and a random 4 digit string.",
```

File: docs/openapi/components/schemas/credentials/FSMATransformingCTECertificate.yml [hunk 4]
```diff
@@ -131,51 +139,51 @@ example: |-
...
         ]
       },
       "foodProduced": {
-        "type": "FSMAProduct",
+        "type": ["FSMAProduct"],
         "traceabilityLot": {
-          "type": "FSMATraceabilityLot",
+          "type": ["FSMATraceabilityLot"],
           "lotCode": "638854954425-FP-PP16-061221-6H08",
           "lotCodeAssignmentMethod": "Lot codes are assigned based on the product ID, location, process date and a randomly generated string.",
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/FSMATransformingCTECertificate.yml:188]:
```suggestion
          "lotCodeAssignmentMethod": "Lot codes are assigned based on the product ID, location, process date, and a randomly generated string.",
```

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**